service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/vm/EnableVmLoggingStep.java

```diff
@@ -78,7 +79,7 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
   }
 
   @WithSpan
-  private void createDataCollectionRuleAssociation(
+  void createDataCollectionRuleAssociation(
```

Comment:
add `@VisibleForTesting` annotation to this as well as `createExtension` and `removeExtension`

Reply:
Added `com.google.common.annotations.VisibleForTesting` to these three methods. Thanks!